About/FAQ.md

```diff
@@ -161,26 +161,48 @@ Unlike [Tails](https://tails.boum.org/), which aims to be a stateless OS that
-the one authentication attempt.  Uniqueness is based on a counter which is
+one authentication attempt. Uniqueness is based on a counter which is
 incremented each authentication attempt.
 
-TOTP (Time-based One-time Password algorithm) is an extension of HOTP but
-replaces the counter with time.  Because of latency, both network and human,
+**TOTP (Time-based One-time Password algorithm)** is an extension of HOTP but
+replaces the counter with time. Because of latency, both network and human,
 and unsynchronised clocks, the one-time password must validate over a range of
 times between the authenticator and the user. Here, time is
 downsampled into larger durations (e.g., 30 seconds) to allow for validity
 between the parties.
 
-Secuirty wise, HOTP is more susceptible to brute force attacks without
-throttling or limiting the number of failed attempted while TOTP is susceptible
+Security wise, HOTP is more susceptible to brute force attacks without
+throttling or limiting the number of failed attempts while TOTP is susceptible
 to phishing attacks and requires a user to enter the code within a given time
 period.
 
+### Reverse HOTP (Heads Firmware Verification)
+Heads uses a specialized **reverse HOTP** protocol for firmware verification. Unlike standard HOTP where the client proves identity to a server, reverse HOTP allows the firmware to prove its integrity to the user through a USB Security dongle. This requires specific firmware support that is only available in certain USB Security dongles (see the [Prerequisites compatibility table](/Installing-and-Configuring/Prerequisites#compatible-usb-security-dongles) for current supported models).
+
+**Important**: Many USB Security dongles (including YubiKey 5 Series) support standard HOTP/TOTP but do NOT support the reverse HOTP protocol that Heads requires.
+
+### TPMTOTP (Heads Attestation)
+**TPMTOTP** is generated by Heads itself using TPM measurements of the firmware. It displays a code on the screen that you can verify against your phone's authenticator app to ensure the firmware hasn't been tampered with. This doesn't require any special USB Security dongle, but it does require:
+
+- A smartphone or device with an authenticator app
+- **Proper time synchronization**: Both your device and the smartphone must have accurate time (preferably UTC/GMT). Time drift of more than 30 seconds can cause TOTP codes to not match
+- For non-networked systems: Manual time setting may be required, as automatic time synchronization won't be available
+
+**Note**: TPMTOTP time-based verification can be challenging on systems that haven't been powered on for extended periods or lack network time synchronization. In such cases, USB Security dongles with reverse HOTP provide more reliable verification since they don't depend on time synchronization.
+
+### What each component does in Heads:
+- **USB Security dongle**: Stores your private key for signing `/boot` contents + provides reverse HOTP verification
```

Comment:
"Stores your private key" -- this is where the OpenPGP requirement comes in, right? Would be good to mention OpenPGP by name here.